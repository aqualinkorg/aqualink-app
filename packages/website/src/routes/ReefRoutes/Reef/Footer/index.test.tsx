import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import ReefFooter from ".";

test("renders as expected", () => {
  const { container } = render(
    <Router>
      <ReefFooter />
    </Router>
  );
  expect(container).toMatchSnapshot();
});
