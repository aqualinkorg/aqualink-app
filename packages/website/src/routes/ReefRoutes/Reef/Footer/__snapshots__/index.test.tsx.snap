// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary Footer-appBar-1 MuiPaper-elevation4"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10"
        >
          <h4
            class="MuiTypography-root MuiTypography-h4"
          >
            Aqua
          </h4>
          <h4
            class="MuiTypography-root MuiTypography-h4"
            style="color: rgb(138, 198, 222);"
          >
            link
          </h4>
        </div>
      </div>
    </div>
  </header>
</div>
`;
